tests/e2e/support/helpers.js

```diff
@@ -245,7 +245,7 @@ function assertDomMatchesFixture(pNum, fixture) {
     } else {
       cy.get('[data-opponent-hand-card]').should('not.exist');
     }
-    if (playerHasGlasses) {
+    if (playerHasGlasses || spectating) {
```

Comment:
If we're asserting during spectate mode we expect the opponent hand cards to be visible